token-lending/program/src/instruction.rs

```diff
@@ -161,15 +161,15 @@ pub enum LendingInstruction {
     RefreshObligation,
 
     // 8
-    /// Deposit collateral to an obligation. Requires a refreshed reserve.
+    /// Deposit collateral to an obligation.
     ///
     /// Accounts expected by this instruction:
     ///
     ///   0. `[writable]` Source collateral token account.
     ///                     Minted by deposit reserve collateral mint.
     ///                     $authority can transfer $collateral_amount.
     ///   1. `[writable]` Destination deposit reserve collateral supply SPL Token account.
-    ///   2. `[]` Deposit reserve account - refreshed.
+    ///   2. `[writable]` Deposit reserve account.
```

Comment:
Nit: Do we need to make the same comment change for  `RedeemReserveCollateral` given it no longer needs to be refreshed?

Reply:
ah yeh prob should change comment (the reserve over there was alright writable so just removing the "refreshed"